### PR TITLE
Fix support for BlobType

### PR DESCRIPTION
**Pull request by yanivshaked**
This PR fixes issue #1600

### CODE REVIEW

File: drift/lib/src/runtime/types/sql_types.dart [hunk 1]
```diff
@@ -168,7 +168,8 @@ class BlobType extends SqlType<Uint8List> {
 
   @override
   Uint8List? mapFromDatabaseResponse(dynamic response) {
-    return response as Uint8List?;
+    var list = response.codeUnits;
+    return Uint8List.fromList(list);
```
Comment (simolus3) [inline drift/lib/src/runtime/types/sql_types.dart:172]: `response` should really be a `Uint8List` already, not a string. Now this method only works when given a string, but we can actually store blobs in the database.

Comment (yanivshaked) [reply drift/lib/src/runtime/types/sql_types.dart:172]: Correct. To make it more robust, how about:
```
  @override
  Uint8List? mapFromDatabaseResponse(dynamic response) {
    if (response is String) {
      var list = response.codeUnits;
      return Uint8List.fromList(list);
    }
    return response as Uint8List?;
  }
```

File: drift/lib/src/runtime/types/sql_types.dart [hunk 2]
```diff
@@ -168,6 +168,10 @@ class BlobType extends SqlType<Uint8List> {
 
   @override
   Uint8List? mapFromDatabaseResponse(dynamic response) {
+    if (response is String) {
```
Comment (simolus3) [inline drift/lib/src/runtime/types/sql_types.dart:171]: Could you also add a test verifying this behavior [here](https://github.com/simolus3/moor/blob/0a2cc40ba5bb0aa78c6192c479ddd1b22bef8ad3/drift/test/types/blob_test.dart#L6)? Just test that `mapFromDatabaseResponse` with a string returns an expected list of bytes.

Comment (yanivshaked) [reply drift/lib/src/runtime/types/sql_types.dart:171]: Sure, will do so later this evening

Comment (yanivshaked) [reply drift/lib/src/runtime/types/sql_types.dart:171]: Done, test added

**Comment by simolus3** [comment]
Looks good, thanks!

**Comment by yanivshaked** [comment]
@simolus3 Thanks, how soon can we expect a version release?

**Comment by simolus3** [comment]
I have released this in 1.2.0. Thanks for your contribution!

**Comment by yanivshaked** [comment]
My pleasure, thank you!!

**Pull request merged**